README.md

```diff
@@ -16,8 +16,11 @@ The local deployment is handled using [cannon](https://usecannon.com), which pro
 
 ## Contracts
 
-| Name | Version | Address | Original Repo | Cannon Package |
-| ---- | ------- | ------- | ------------- | -------------- |
+| Name       | Version | Address                                                                                                               | Original Repo                                                     | Cannon Package                  |
+| ---------- | ------- | --------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------- | ------------------------------- |
+| EntryPoint | 0.6     | [0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789](https://etherscan.io/address/0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789) | https://github.com/eth-infinitism/account-abstraction/tree/v0.6.0 | eth-infinitism-entrypoint:0.6.0 |
+| EntryPoint | 0.7     | [0x0000000071727De22E5E9d8BAf0edAc6f37da032](https://etherscan.io/address/0x0000000071727De22E5E9d8BAf0edAc6f37da032) | https://github.com/eth-infinitism/account-abstraction/tree/v0.7.0 | eth-infinitism-entrypoint:0.7.0 |
```

Comment:
It was hard to verify this address just from the repo code.
Later, I found the address in the 0.7.0 release notes.
Maybe the versions (0.6, 0.7, 0.8) could be links to the release notes?
Then, the links to the original repo may be removed (not sure if the other contract suites are also versioned through GitHub Releases)?

Reply:
The others are not, in general.
It's scary how badly these things are maintained.
I'll add links to the release